_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-tweetstorm-parse.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * REST API endpoint for parsing Tweetstorms out of block content.
+ *
+ * @package Jetpack
+ * @since 9.0.0
+ */
+
+use Automattic\Jetpack\Connection\Client;
+
+/**
+ * Tweetstorm gatherer.
+ *
+ * @since 9.0.0
+ */
+class WPCOM_REST_API_V2_Endpoint_Tweetstorm_Parse extends WP_REST_Controller {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'tweetstorm';
+
+		if ( ! class_exists( 'Jetpack_Tweetstorm_Helper' ) ) {
+			\jetpack_require_lib( 'class-jetpack-tweetstorm-helper' );
+		}
+
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register the route.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/parse',
+			array(
+				'args'                                  => array(
+					'blocks' => array(
+						'description' => __( 'An array of serialised blocks, and editor-specific block information.', 'jetpack' ),
```

Comment:
Nitpick: We should tweak this to the American spelling.
```suggestion
						'description' => __( 'An array of serialized blocks, and editor-specific block information.', 'jetpack' ),
```